### ISSUE TITLE
[Bug]: bufferline does not show up

### ISSUE THREAD

**Issue by ekscentrysytet** [akinsho/bufferline.nvim#723]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Env:
neovim v0.8.1.

I use packer and installed plugin like this:
```lua
use 'akinsho/bufferline.nvim'
```
then in my top-level `init.lua` I setup it like:
```lua
require('bufferline').setup {}
```

After running `:PackerSync` exiting and entering neovim bufferline does not show up.

It only shows up when I trigger
```
:lua require('bufferline').setup {}
```
Same issue was reported [here](https://www.reddit.com/r/neovim/comments/wq6hz7/comment/ikktzfp/). 

### What did you expect to happen?

Expected behaviour:

bufferline shows up with default settings

### Config

```lua
use 'akinsho/bufferline.nvim'
```
then in my top-level `init.lua` I setup it like:
```lua
require('bufferline').setup {}
```

### Additional Information

...

**Comment by ekscentrysytet** [comment]
my nvim config can be observed [here](https://github.com/ekscentrysytet/dotfiles/tree/main/.config/nvim)

**Comment by akinsho** [comment]
@ekscentrysytet this issue is almost certainly something to do with how your config is setup, rather than a bug in this plugin. Unfortunately as a general rule I can't debug individual user's configs to see why the plugin may or may not be loaded. If I did this would likely be the only thing I did with this plugin. In a brief look at your config I can't see what the issue is. I can definitely attest to it working and think other users not reporting this issue is also validates this
